### DISCUSSION TITLE
Application with long initialization does not pass the audit

### DISCUSSION THREAD

**Discussion by jpx100** [GoogleChrome/lighthouse#14938] [Q&A] [answered]
I have application which takes quite long time to initialize, because there is long list of resource to be downloaded before the service worker is installed. But this long time is breaking the audit and the report says that the application is not installable. Is there a way how to increase the timeout before the installable test fails? 
Otherwise the application is OK, and when I manually reduced the list of resources it passed all audits.

Thanks for your advice

Version: Lighthouse 10.0.2
Command to run lighthouse: lighthouse {URL of my application, unfortunately not public} --only-categories pwa --view --verbose
![Screenshot 2023-03-29 at 12-26-04 Lighthouse Report](https://user-images.githubusercontent.com/65395881/228505748-8f6b0e8f-3c3f-43d9-ad06-b5faa6ae506e.png)

**Comment by adamraine** [comment, marked as answer]
You can extend how long Lighthouse waits after the page load event using a [custom config](https://github.com/GoogleChrome/lighthouse/blob/main/docs/configuration.md):

```js
export default {
  extends: 'lighthouse:default',
  settings: {
    // This will control how long Lighthouse needs to wait after the load event
    pauseAfterLoadMs: 10_000,
  }
}
```

Feel free to tune the value of `pauseAfterLoadMs` yourself, I just chose 10s as a starting point.

**Reply by jpx100** [reply]
It works fine, thanks!
